tests/integration_tests/test_byz_node_sending_different_views.py

```diff
@@ -0,0 +1,102 @@
+"""
+Case 7.1
+Byzantine node 0 tells some nodes that its view is (1,1) and others that it is (2,2).
```

Comment:
Might be nice to describe the target state a bit here as well, i.e. that all nodes with `ID > 0` reach `{"current": 1, "next": 1}` and why that is the target state. Think we should document all integration tests more thoroughly as well, adding ticket for that

Reply:
You are correct, updated the description in both tests.